subsys/bluetooth/controller/hci/hci_internal.h

```diff
@@ -40,3 +28,9 @@ int hci_acl_handle(struct net_buf *acl, struct net_buf **evt);
 void hci_acl_encode(struct node_rx_pdu *node_rx, struct net_buf *buf);
 void hci_num_cmplt_encode(struct net_buf *buf, u16_t handle, u8_t num);
 #endif
+int hci_vendor_cmd_handle(u16_t ocf, struct net_buf *cmd,
+			  struct net_buf **evt);
+int hci_vendor_cmd_handle_common(u16_t ocf, struct net_buf *cmd,
+			     struct net_buf **evt);
+void *cmd_complete(struct net_buf **buf, u8_t plen);
```

Comment:
cmd_complete and evt_create should have hci_ as well.

Reply:
OK